### ISSUE TITLE
[Bug] Trackpad/Mouse Support

### ISSUE THREAD

**Issue by JagandeepBrar** [CometTools/LunaSea#206]
Currently, LunaSea (and Flutter in general) does not support iPadOS’s newly introduced mouse and trackpad support. Clicking around the application works, but scrolling actions do not work.

This has to be deferred though, as this is a problem with the Flutter framework itself and not something that I can fix myself.

Just creating an issue to say that this issue is known, and will be fixed as soon as possible.

**Comment by kalanihelekunihi** [comment]
I can confirm this issue remains on Big Sur on ARM. On that platform, scrolling is only possible by clicking and dragging.

The issue is ultimately upstream within Flutter, as seen here: https://github.com/flutter/flutter/issues/55809
As a P5 incident, it is treated as a feature enhancement request rather than a bug.

Hopefully with macOS leveraging the same code, it can elevate due to unintended and non-intuitive behavior.

In the mean time, there are options to work around this using UIPanGestureRecognizer, however this may not be an especially elegant solution as upstream changes will likely break.

**Comment by JagandeepBrar** [comment]
@kalanihelekunihi Thanks for checking on the new machines, I really hope that they prioritize the issue as it will become more and more common with more magic keyboards and the new ARM machines. It appears that P5 is the default priority for new feature requests, and there is a pending PR to add in basic support: https://github.com/flutter/engine/pull/19965

And yeah, ideally I don't want to implement a workaround and am waiting on the Flutter team to merge the changes themselves.

**Comment by JagandeepBrar** [comment]
Moving this request to the feedback board, any progress will be updated there: https://feedback.lunasea.app/b/User-Experience/p/keyboard-support
